### ISSUE TITLE
Computing just logits/log prob without getting loss from Adaptive Softmax

### ISSUE THREAD

**Issue by tonyhqanguyen** [kimiyoung/transformer-xl#98]
Can anyone point out how I can get only the log probability (for actual decoding) from the Adaptive Softmax without getting the loss. The forward method in the class in this repo only gives the loss, but not the log probs. 

Thanks
